man/git-get.md

```diff
@@ -0,0 +1,34 @@
+git-get(1) -- Clone a Git repository under a directory
+=================================================
+
+## SYNOPSIS
+
+`git-get`
+
+## DESCRIPTION
+
+  Clones a Git repository under the directory specified by the environment variable `GIT_EXTRA_DEFAULT_CLONE_PATH`
+
+## EXAMPLES
+
+    $ GIT_EXTRA_DEFAULT_CLONE_PATH="$HOME/some-dir" git-get 'https://github.com/hyperupcall/bake'
```

Comment:
It seems that this feature is not much convenient than `git clone 'https://github.com/hyperupcall/bake' $GIT_EXTRA_DEFAULT_CLONE_PATH/bake`. Could you provide some real-world use cases for this tool?

Reply:
The linked issue did suggest making a PR, but this change additionally significantly reduces typing.

On some keyboards (like mine), the `~` is either touch to reach, or requires an extra button (Fn+Shift for some 80%s). `$HOME` is also quite long to type. Besides that, this is a good solution when cloning frequently. It's common to clone repositories (maybe for just sifting through or quickly reading code) to a certain directory, such as `~/Programming/git`. I myself have about 50 Git repositories cloned there. Having an actual variable like `$GIT_CLONE_DIR` is hard to remember and again is on the longer side.

Reply:
I see